### PR TITLE
Lint build files with ktlint

### PR DESCRIPTION
**Pull request by JLLeitschuh**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -23,6 +23,18 @@ plugins {
     jacoco
     id("edu.wpi.first.wpilib.versioning.WPILibVersioningPlugin") version "1.6"
     id("com.github.johnrengelman.shadow") version "2.0.1"
+    id("com.diffplug.gradle.spotless") version "3.5.1"
+}
+
+allprojects {
+    apply {
+        plugin("com.diffplug.gradle.spotless")
```
Comment (SamCarlberg) [inline build.gradle.kts]: You should leave a comment for what this plugin does and why we use it. For maintainability

Comment (JLLeitschuh) [inline build.gradle.kts]: Really?
http://bfy.tw/DNA1

I mean sure if you think that it's necessary.

Comment (SamCarlberg) [inline build.gradle.kts]: It's good to have to future maintainers can easily know what's going

Comment (JLLeitschuh) [inline build.gradle.kts]: Okay, sure.

Comment (JLLeitschuh) [inline build.gradle.kts]: Fixed, see below

**Comment by JLLeitschuh** [comment]
I'm asking various developers why they think Windows is failing.

**Comment by JLLeitschuh** [comment]
The `.gitattributes` file fixed the problem for windows.
https://github.com/diffplug/spotless/blob/master/plugin-gradle/README.md#line-endings-and-encodings-invisible-stuff

File: .appveyor.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ build_script:
   - gradlew.bat shadowJar --no-daemon --stacktrace
 
 test_script:
-  - gradlew.bat check --stacktrace -PlogTests -Pheadless
+  - gradlew.bat spotlessCheck check --stacktrace --info -PlogTests -Pheadless
```
Comment (AustinShalit) [inline .appveyor.yml]: We can remove the info flag now.

Comment (JLLeitschuh) [inline .appveyor.yml]: Okay will do

**Comment by JLLeitschuh** [comment]
I'm on vacation. I'll take a look on monday.

**Comment by nightpool** [comment]
have a good trip!

**Comment by JLLeitschuh** [comment]
LGTM!

**Pull request merged**